### PR TITLE
Bug 1069221 - Fixed discrete visibility for screen readers while warnings are displayed (fixed for v3.0)

### PR DESCRIPTION
**Pull request by funktr0n**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[FunkTron FunkTron](https://github.com/FunkTron) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6ca999466fa8)

File: apps/fm/js/fm.js [hunk 1]
```diff
@@ -183,12 +183,15 @@ function updatePowerUI() {
...
-function updateAntennaUI() {
-  $('antenna-warning').hidden = mozFMRadio.antennaAvailable;
-}
-
-function updateAirplaneModeUI() {
+function updateWarningModeUI() {
   $('airplane-mode-warning').hidden = !airplaneModeEnabled;
+  $('antenna-warning').hidden = mozFMRadio.antennaAvailable || 
+    airplaneModeEnabled;
+  if(airplaneModeEnabled || !mozFMRadio.antennaAvailable){
```
Comment (yzen) [inline apps/fm/js/fm.js]: ```$('container').classList.toggle('hidden-block', airplaneModeEnabled || !mozFMRadio.antennaAvailable);```

Comment (FunkTron) [inline apps/fm/js/fm.js]: Oh right, forgot about the true/false flag on "toggle"

**Comment by try-server-hook** [comment]
[FunkTron FunkTron](https://github.com/FunkTron) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dc47a6f0c3af)

**Pull request closed without merging**